### PR TITLE
Update build script. Add TRAVIS CI

### LINKED ISSUE #5: Create build script
We need to create a working cross-platform build script to automate the installation of the set-up so that the python wrapper works. 

This will at the time of writing include:
* Installing miniconda
* Creating the conda environment
* building PyCall against that environment
* Updating the python path

### PR DESCRIPTION
**Pull request by AlexRobson**
Fixes #5 #6 

Can probably switch to using their provided anaconda distirubiton through `Conda.add("data_pipeline_api"; channel="scottishcovidresponse")`, but i believe it requires the MR i have added here: 

https://github.com/ScottishCovidResponse/data_pipeline_api/pull/26

### CODE REVIEW

**Comment by AlexRobson** [comment]
Fixes #5 #6

File: deps/build.jl
Comment (sdl1) [inline deps/build.jl]: Might need to check something else here (or leave out the check) since we're not directly cloning the directory?

File: Project.toml
Comment (sdl1) [inline Project.toml:4]:
```suggestion
version = "0.2.1"
```

File: deps/build.jl
Comment (sdl1) [inline deps/build.jl]: Should we wait for that PR to be merged before this one? As I guess things might break otherwise if that branch gets deleted.

**Review by sdl1 — CHANGES_REQUESTED** [review]
We should change the README installation instructions as well

**Comment by sdl1** [comment]
I can build fine, but when I try to use the package I get an error:
```
(SimulationData) pkg> build SimulationData
   Building Conda ─────────→ `~/.julia/packages/Conda/3rPhK/deps/build.log`
   Building HDF5 ──────────→ `~/.julia/packages/HDF5/pAi1D/deps/build.log`
   Building PyCall ────────→ `~/.julia/packages/PyCall/zqDXB/deps/build.log`
   Building SimulationData → `~/github/SimulationData.jl/deps/build.log`

julia> using SimulationData
[ Info: Precompiling SimulationData [3d44aec0-db1b-416b-9784-2428c815ea7f]
[ Info: Installing numpy via the Conda numpy package...
[ Info: Running `conda install -y numpy` in root environment
Collecting package metadata (current_repodata.json): done
Solving environment: done

# All requested packages already installed.

ERROR: LoadError: LoadError: InitError: PyError (PyImport_ImportModule

The Python package numpy could not be found by pyimport. Usually this means
that you did not install numpy in the Python version being used by PyCall.

PyCall is currently configured to use the Julia-specific Python distribution
installed by the Conda.jl package.  To install the numpy module, you can
use `pyimport_conda("numpy", PKG)`, where PKG is the Anaconda
package the contains the module numpy, or alternatively you can use the
Conda package directly (via `using Conda` followed by `Conda.add` etcetera).

Alternatively, if you want to use a different Python distribution on your
system, such as a system-wide Python (as opposed to the Julia-specific Python),
you can re-configure PyCall with that Python.   As explained in the PyCall
documentation, set ENV["PYTHON"] to the path/name of the python executable
you want to use, run Pkg.build("PyCall"), and re-launch Julia.

) <class 'ImportError'>
ImportError("Something is wrong with the numpy installation. While importing we detected an older version of numpy in ['/Users/sean/.julia/conda/3/lib/python3.6/site-packages/numpy']. One method of fixing this is to repeatedly uninstall numpy until none is found, then reinstall this version.",)
  File "/Users/sean/.julia/conda/3/lib/python3.6/site-packages/numpy/__init__.py", line 142, in <module>
    from . import core
  File "/Users/sean/.julia/conda/3/lib/python3.6/site-packages/numpy/core/__init__.py", line 70, in <module>
    raise ImportError(msg.format(path))
```

File: deps/build.jl
Comment (AlexRobson) [reply deps/build.jl]: Yeah that'll be best practice. I submitted this MR with this in to demonstrate that it works. The issue of running this without that should be included in https://github.com/ScottishCovidResponse/data_pipeline_api/pull/26

**Comment by AlexRobson** [comment]
> We should change the README installation instructions as well

README updated.

File: deps/build.jl
Comment (sdl1) [reply deps/build.jl]: Looks like that was just merged, so this PR should be good to go after this install command is changed

Comment (AlexRobson) [reply deps/build.jl]: Thanks. I've made this change, so if this passes CI this should be GTG

**Pull request merged**
